app/controllers/schools/advice/electricity_out_of_hours_controller.rb

```diff
@@ -1,6 +1,9 @@
 module Schools
   module Advice
     class ElectricityOutOfHoursController < AdviceBaseController
```

Comment:
I believe we need at least a years worth of data to run this analysis - that seems to be what the old advice page required.

So there needs to be some checking for enough data in the controller, see recent improvements to AdviceBase